datamap/bloom/pom.xml

```diff
@@ -23,6 +23,18 @@
       <artifactId>carbondata-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.carbondata</groupId>
+      <artifactId>carbondata-processing</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <!--note: guava 14.0.1 is omitted during assembly.
+    The compile scope here is for building and running test-->
```

Comment:
you have not added compiler scope

Reply:
Oh, this line is to be removed. It is used for guava-cache previously. will fix it